exe/etcd-env

```diff
@@ -40,10 +44,11 @@ def fetch(uri_str, limit = 10)
 end
 
 if __FILE__ == $PROGRAM_NAME
-  key, *commands = ARGV
+  key_str, *commands = ARGV.join(" ").split(" -- ")
+  keys = key_str.split(" ")
```

Comment:
Converting to a string and back is a bit messy. Something like this would be cleaner:

```ruby
sep = ARGV.index '--'
if sep
  keys = ARGV[0...sep]
  commands = ARGV[(sep + 1)..-1]
else
  keys = ARGV.take(1)
  commands = ARGV.drop(1)
end
```